frog/imports/ui/GraphEditor/SidePanel/fileConvert.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React, { Component } from 'react';
+import Form from 'react-jsonschema-form';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+import { Uploads } from '/imports/api/uploads';
```

Comment:
I'm surprised that the path `/imports/api/uploads` works! I was expecting `../../../api/uploads`